main.js

```diff
@@ -121,7 +122,8 @@
 
                 record.pin = L.marker([record.Y, record.X], { icon: CITY_POLICE_ICON });
                 record.pin.addTo(map)
-                    .bindPopup(`${record.OFFENSES}`);
+                
+                    .bindPopup(JSONtoTable(record));
```

Comment:
I don't think we want to loop over all of the fields/values in the results. Some really are not user friendly, and we don't want to rely on making them friendly in the dataset itself. So we will make it a responsibility of each dataset parsing to actually designate the fields worth viewing and a friendly label for that field.

For example, I might only be interested in the time a police incident occurred and what type of incident it was. I'll create a new sub-object property in the `record` for this description. For example, I will only include those two fields and  `incidentDateOccurance` isn't friendly, so I'll change the label of that:

```
record.description = [{
    label: "Time",
    value: record.incidentTime //or whatever it is
},
{
    label: "Type",
    value: record.incidentType
},
{
    //etc.
}]
```

We'll then pass _this_ sub-obj into that function, and bind the popup to HTML string generated.

You're not responsible for creating these sub-objects right now (unless you want to!); just making sure the generate HTML function works.